src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -887,7 +888,7 @@ private async Task ScheduleUpdateForLater(InProgressPullRequest pr, Subscription
         _logger.LogInformation("PR {url} for subscription {subscriptionId} cannot be updated at this time. Deferring update..", pr.Url, update.SubscriptionId);
         await _pullRequestUpdateReminders.SetReminderAsync(update, DefaultReminderDelay, isCodeFlow);
         await _pullRequestCheckReminders.UnsetReminderAsync(isCodeFlow);
-        pr.NextCommitToProcess = update.SourceSha;
+        pr.NextBuildsToProcess[update.SourceRepo] = update.BuildId;
```

Comment:
Can the key be the subscription ID instead?

Reply:
sure, replaced